ext/opentelemetry-ext-wsgi/src/opentelemetry/ext/wsgi/__init__.py

```diff
@@ -87,8 +85,11 @@ def __call__(self, environ, start_response):
 
         tracer = trace.tracer()
         path_info = environ["PATH_INFO"] or "/"
+        parent_span = propagator.get_global_propagator().extract(
```

Comment:
I'd name that parent_context (because it is no span but a context).

Reply:
is that the right call? the trace/__init__.py defines the type as "ParentSpan"

Reply:
I'd argue that either both should be called ParentContext, or ParentSpan.